packages/cli/src/imagerenderer.ts

```diff
@@ -0,0 +1,149 @@
+import sharp from "sharp";
+
+export type TerminalType = "kitty" | "iterm2" | "none";
+
+const KITTY_IDENTIFIERS: string[] = [
+  "kitty",
+  "wezterm",
+  "konsole",
+  "warp",
+  "wayst",
+  "st",
+];
+
+type KittyCommand = Record<string, string | number>;
+
+export function detectTerminalCapabilities(): TerminalType {
+  const termProgram = (Deno.env.get("TERM_PROGRAM") || "").toLowerCase();
+
+  for (const id of KITTY_IDENTIFIERS) {
+    if (termProgram === id) {
+      return "kitty";
+    }
+  }
```

Comment:
You could utilize [`Array.prototype.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) method.

```suggestion
  if (KITTY_IDENTIFIERS.includes(termProgram)) return "kitty";
```